blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubEnterpriseScm.java

```diff
@@ -36,7 +35,7 @@ public GithubEnterpriseScm(Reachable parent) {
         String apiUri = getCustomApiUri();
 
         // NOTE: GithubEnterpriseScm requires that the apiUri be specified
-        if (StringUtils.isEmpty(apiUri)) {
+        if (apiUri == null || apiUri.isEmpty()) {
```

Comment:
is this the same?   does the upstream not treat a blank (all whitespace) string as empty?
(the java method is the string has length zero)

Reply:
yes it is :-)

https://www.javadoc.io/doc/org.parboiled/parboiled-core/0.11.0/org/parboiled/common/StringUtils.html#isEmpty(java.lang.String)

Reply:
Yep, I checked, I should have added a comment on the pr